gulpfile.js

```diff
@@ -88,7 +88,17 @@ gulp.task('copy-app-common', () => {
 
     // Polyfills.
     gulp.src('./node_modules/whatwg-fetch/fetch.js')
-      .pipe(rename('js/polyfills/fetch.js'))
+      .pipe(rename('js/polyfills/fetch.js')),
+
+    gulp.src('./app/js/third_party/webrtc-adapter.js')
```

Comment:
Can you place these files in the `components` folder instead `third_party` (see above with lodash and moment)?